src/modules/create-market/components/create-market-form-outcome/create-market-form-outcome.jsx

```diff
@@ -209,6 +210,14 @@ export default class CreateMarketOutcome extends Component {
       updatedMarket.tickSize = value
     }
 
+    // Make sure scalarBigNum, scalarSmallNum, & numTicksBigNum are all BigNumbers
+    if (BigNumber.isBigNumber(scalarBigNum) && BigNumber.isBigNumber(scalarSmallNum) && BigNumber.isBigNumber(numTicksBigNum)) {
+      // Always check if (maxPrice - minPrice) / precision is an even number
+      if ((scalarBigNum.minus(scalarSmallNum).div(numTicksBigNum)).mod(2).toString() !== '0') {
```

Comment:
This looks like validation that should/could be performed in the switch, is there a reason we do this separately from the above switch?

Reply:
I originally tried putting this check in the switch statement above, but the error message needs to get updated as the user is typing in the "Precision" textbox, as well as the "Min Value"/"Max Value" textboxes, since all of these values are used to determine the number of ticks.